Create ui-c8y-1024-7-2-fix-data-explorer-export-modal-spacing-for-am-pm-time-selector.md (#4955)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1024-7-2-fix-data-explorer-export-modal-spacing-for-am-pm-time-selector.md

---
date: ""

> title: Fix data explorer export modal spacing for AM/PM time selector

Reviewer: ```suggestion
title: Fixed spacing for AM/PM time selector in the data explorer export modal
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-67005
version: 1024.7.2
---

> Fixed misaligned spacing around the AM/PM time selector in the data explorer export modal.

Reviewer: ```suggestion
The spacing around the AM/PM time selector in the data explorer export modal was misaligned. This issue has been fixed.
```

---
date: ""
title: fix data explorer export modal spacing for AM/PM time selector (#12463) [GRAFT][release/cd] (#12523)
product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-67005
version: 1024.7.2
---
# Backport

This will backport the following commits from `develop` to `release/cd`:
- [fix(Web SDK): [MTM-67005] fix data explorer export modal spacing for
AM/PM time selector
(#12463)](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/12463)

<!--- Backport version: unknown -->

### Questions ?
Please refer to the [Backport tool
documentation](https://github.com/sorenlouv/backport)

[MTM-67005]:
https://cumulocity.atlassian.net/browse/MTM-67005?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

> Co-authored-by: Carlos Ceia <carlos.ceia@cumulocity.com>

Reviewer 2: ```suggestion
Fixed misaligned spacing around the AM/PM time selector in the data explorer export modal.
```
